src/frontend/screens/Game/GamePage/components/HLTBInline.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useContext, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import GameContext from '../../GameContext'
+import { Speed, ExpandMore } from '@mui/icons-material'
+import { Accordion, AccordionSummary, AccordionDetails } from '@mui/material'
+import HowLongToBeat from 'frontend/components/UI/WikiGameInfo/components/HowLongToBeat'
+
+const HLTBInline = () => {
```

Comment:
maybe we can include the different logic from this inside the current `HLTB` component? and depending on the `experimentalFeatures.enableNewDesign` value? I did that already in the Scores component to make it inline

The reason is that at some point we are going to remove the old design and we'll have to remember to remove one of these HLTB components. If we just add some conditional code in the current HLTB component it's going to be easier to clean in the future.

Reply:
Good point! Only downside is that `isExpanded` will be present regardless of the design since react hook must be at component top level.

Reply:
yeah, not a big deal though, it's just a state that won't be used in the old design, but I wouldn't worry about that

Reply:
Fixed